ui/src/main/webapp/src/app/components/reports/application-index/aggregated-statistics.service.ts

```diff
@@ -0,0 +1,35 @@
+import {Injectable} from "@angular/core";
+import {Http} from "@angular/http";
+import {Observable} from "rxjs";
+import {AbstractService} from "../../../services/abtract.service";
+import {Constants} from "../../../constants";
+import {IssueCategoryModel} from "../../../generated/tsModels/IssueCategoryModel";
+
+@Injectable()
+export class AggregatedStatisticsService extends AbstractService {
+    private static AGGREGATED_MANDATORY_INCIDENTS_URL = 'aggregatedstats/aggregatedMandatoryIncidents';
+    private static AGGREGATED_CATEGORIES_URL = 'aggregatedstats/aggregatedCategories';
+    private static AGGREGATED_JAVA_PACKAGES_URL = 'aggregatedstats/aggregatedJavaPackages';
+
+    constructor(private _http: Http) {
+        super();
+    }
+
+    getAggregatedCategories(executionId: number): Observable<any[]> {
```

Comment:
Well, I have there originally something like Map<IssueCategoryModel, number> https://github.com/windup/windup-web/pull/191/commits/4772effb06cf1eb1217497e1795d10b2166b8337
But on @klinki suggestions I chose the typeless return types. I agree it is not nice, but basically I don't need everything what is original models provided. But it is true it would be better to have there a typescript object instead.